src/Automation/SnapshotCommand.cs

```diff
@@ -88,29 +73,11 @@ public static SnapshotCommandResult Execute(Config config, IOutputFileHelper out
                                 if (locationHelper.IsSarifExtension())
                                     // SaveAction.SaveSarifFile(outputFileHelper.GetNewSarifFilePath(), ec2.Id, !locationHelper.IsAllOption());
```

Comment:
Should have one more indent
```suggestion
                                        // SaveAction.SaveSarifFile(outputFileHelper.GetNewSarifFilePath(), ec2.Id, !locationHelper.IsAllOption());
```